cmd/kubelet/app/server.go

```diff
@@ -252,7 +252,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 			if err := checkPermissions(); err != nil {
 				klog.ErrorS(err, "kubelet running with insufficient permissions")
 			}
-			// set up signal context here in order to be reused by kubelet and docker shim
+			// set up signal context here in order to be reused by kubelet
 			ctx := genericapiserver.SetupSignalContext()
```

Comment:
now that we don't reuse the context we can actually move this to line 266, to keep it more closely scoped to the place it's used.

Reply:
Hi, @endocrimes, thanks for review, I have applied all suggestions. Could you take a look again? thanks.